.buildkite/shared-pipeline-vars

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# This file is `source`'d before calling `buildkite-agent pipeline upload`, and can be used
+# to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.
+
+# The ~> modifier is not currently used, but we check for it just in case
+XCODE_VERSION=$(sed -E -n 's/^(~> )?(.*)/xcode-\2/p' .xcode-version)
+CI_TOOLKIT_PLUGIN_VERSION='mokagio/windows-utils'
```

Comment:
This will eventually land in the CI Toolkit `main`. I'll followup once that happens. There are some issues on that end with how the tests are written.